datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx

```diff
@@ -130,6 +132,7 @@ export const EmbeddedListSearch = ({
     useGetDownloadSearchResults = useDownloadScrollAcrossEntitiesSearchResults,
     shouldRefetch,
     resetShouldRefetch,
+    applyView,
```

Comment:
let's do applyView = false here

this will make it so the value of the parameter is false by default!